app/main.js

```diff
@@ -67,9 +67,8 @@ function createTrayIcon () {
   } else {
     appIcon = new Tray(iconFolder + '/stretchly_18x18.png')
   }
-  appIcon.setToolTip('stretchly - break time reminder app')
-  isPaused = false
   appIcon.setContextMenu(getTrayMenu())
+  toolTipUpdater = setInterval(updateToolTip, 10000)
```

Comment:
> Maybe also create function for tool tip updater? Might be more code, but easier to understand what happens when app starts. What do you think?

As in move this line into it's own function, as an additional `app.on('ready')` call?